docs/specs/bug_and_feedback_submission.md

```diff
@@ -48,9 +48,40 @@ When putting an object into a bucket [the client must give it a key](https://doc
 
 This opens a possible attack vector for incurring large bills to SatoshiLabs by an attacker uploading large amounts of data to the buckets. This can't be 100% prevented as the API must remain open because Suite's users don't authenticate in any way. A rate limiting could prevent this but that would require some kind of application layer in between the clients and the S3. Instead, for start, we'll start by setting up a [CloudWatch alarm](https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html) that will warn us if daily uploads exceed some threshold.
 
+## Data
+This section describes what data are sent in the query string. We start with smaller set of fields that can all fit into one level of nesting so that we don't have to decide [which encoding convention](https://stackoverflow.com/a/9547490/5698865) to use for nested fields. More 
+
+### Bugs
+- `description` The textual description of the bug written by the user.
+- `path` Current url path. Corresponds to the "category" in the Designs.
+- `screenshot` (optional) UUID of linked image.
+
+### Feedback
+- `rating` 1-5, 1 is best, 5 is worst.
+- `description` (optional) User's impressions of the app.
+- `screenshot` (optional) UUID of linked image.
+
+### Common
+The fields sent with both bugs and feedback. All device-related fields are optional because the can be no device connected. Most of them can be reused from the [GitHub Issue feature](https://github.com/trezor/trezor-suite/blob/78736782fbd6556f536fcbc989ba15cf0b367555/packages/suite/src/services/github.ts#L36).
```

Comment:
the -> there

But btw that is not true because you can't access the app until there is a Trezor connected (until we do #3729). Or will one be able to open the right Learn & Discover menu also on the Connect your Trezor page?

Reply:
We'd like to support the option to access App & Coin Settings without connected device in the next sprint #3729, including the Learn & Discover.